docs/developer/community/zero-downtime-migrations.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: Zero downtime migrations
+---
+
+## The problem
+
+From Saleor 3.13 version forward, we provide a way to ensure we can safely migrate the database without interrupting the work of already running web workers.
+
+## Solution
+
+To achieve the goal we need to ensure that:
+
+- Old code is compatible with new database schema or new database schema is compatible with old code.
+- Migrations don't lock tables, rows etc. for no longer than one second.
+
+## Writing migrations
```

Comment:
Who is the intended audience of this document?

If the audience is Saleor users, this section does not belong here as we don't recommend that anyone forks Saleor (they would be unable to upgrade).

If the audience is developers, the section on running migrations does not belong here, as it should be part of self-hosting.

Reply:
The idea was actually to provide documentation/guide for developers and users as well. This way we can easily provide link in PR's. As well anyone interested of topic can check how migrations works and how to execute them.

I was also thinking of maybe splitting this document, but at the end decided that with one document is easier to find all relevant information.

Maybe I can just provide more links to this document in proper sections?